packages/shared/lib/core/account/actions/setNextSelectedAccount.ts

```
@@ -0,0 +1,13 @@
+import { selectedAccount, setSelectedAccount } from '@core/account'
+import { nonHiddenActiveAccounts } from '@core/profile'
+import { get } from 'svelte/store'
+
+export function setNextSelectedAccount(): void {
+    const account = get(selectedAccount)
+    const otherAccounts = get(nonHiddenActiveAccounts)
+
+    if (account.hidden) {
```

we should add here ` && otherAccounts.length > 0`.

---

We should throw an error if thats the case, as we should not allow the app to get into this position.

---

So we should disable the "hide" button  if we only have 1 account left?